.github/workflows/trivy-docker-hub-scan.yml

```
@@ -0,0 +1,34 @@
+name: "Trivy vulnerability scanner for Docker Hub Image"
+
+on:
+  workflow_dispatch: # Trigger manually
+  push:
+  pull_request:
```

```suggestion
    inputs:
      image-tag:
        required: false
        type: string
  workflow_call: # Trigger by another workflow
    inputs:
      image-tag:
        required: true
        type: string
      ref:
        type: string
      sha:
        type: string
```
This workflow should only be triggered after the image was built and published to Docker Hub. Otherwise the scan will be using an outdated version of the `latest` image. 
Please also add the trigger to `irs-build.yml` (underneath the job `build_images`)
```
  trigger-trivy-image-scan:
    if: >-
      github.event_name != 'pull_request'
    needs:
      - build_images
    uses: ./.github/workflows/trivy-docker-hub-scan.yml
    with:
      image-tag: ${{ needs.build_images.outputs.image_tag }}
      ref: ${{ github.ref }}
      sha: ${{ github.sha }}
```